pkg/installer/generateconfig.go

```diff
@@ -110,6 +110,12 @@ func (m *manager) generateInstallConfig(ctx context.Context) (*installconfig.Ins
 		SoftwareDefinedNetwork = string(m.oc.Properties.NetworkProfile.SoftwareDefinedNetwork)
 	}
 
+	// determine outbound type based on cluster visibility
+	OutboundType := azuretypes.LoadbalancerOutboundType
```

Comment:
nit - don't use upper-case variable names.  
```suggestion
	outboundType := azuretypes.LoadbalancerOutboundType

Reply:
+1